src/components/List/List.tsx

```diff
@@ -23,7 +23,7 @@ const List: React.FC<listProps> = ({ listItems, columns }) => {
   const examineCell = (cellInfo:any) => {
     if (Array.isArray(cellInfo)) {
       return cellInfo.join(', ')
-    } else if (cellInfo.includes('www.') || cellInfo.includes('https')) {
+    } else if (cellInfo.includes('www.') || cellInfo.includes('http')) {
```

Comment:
I'd like to find a more foolproof way of checking for web-links, or provide a way for the programmer re-using these components to mark which column should be a link.